### PR TITLE
Fix -  3648 - Added "control+enter" hotkey to post comment

### LINKED ISSUE #3648: Add "control-enter" hotkey to post a comment, like on GitHub/Gmail
I write a lot of comments, and sometimes it's nice to be able to hit `control-enter` (or `command-enter` on a Mac) to post the comment. This is a nice convention on GitHub and also for sending an email on Gmail. I think it should be relatively easy to code up using a standard mini library from NPM (https://www.npmjs.com/search?q=keywords:keybinding maybe?) or perhaps some jQuery. 

This will involve a bit of problem solving and creativity -- any help appreciated!

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #3648 

Demo:

ezgif com-video-to-gif 13

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by sk9331657** [comment]
@avsingh999  Please review it.

**Comment by avsingh999** [comment]
@sk9331657 great work 
thanks :)

File: app/views/comments/_form.html.erb [hunk 1]
```diff
@@ -69,9 +69,17 @@
...
+
+      if (e.ctrlKey && e.keyCode == 10) {
+        $("#comment-form .btn-primary").click();
+      }
+    })
+  
     $('#comment-form').bind('ajax:beforeSend', function(event){
       $("#text-input").prop('disabled',true)
       $("#comment-form .btn-primary").button('loading',true);
+
```
Comment (avsingh999) [inline app/views/comments/_form.html.erb]: please remove line 73 and 82

**Comment by sk9331657** [comment]
@avsingh999  Please review.

File: app/views/comments/_form.html.erb [hunk 2]
```diff
@@ -69,11 +69,16 @@
...
+    $('#text-input').bind('keypress',function(e){
+      if (e.ctrlKey && e.keyCode == 10) {
+        $("#comment-form .btn-primary").click();
+      }
+    })
+  
     $('#comment-form').bind('ajax:beforeSend', function(event){
       $("#text-input").prop('disabled',true)
       $("#comment-form .btn-primary").button('loading',true);
     });
```
Comment (avsingh999) [inline app/views/comments/_form.html.erb]: @sk9331657  please add a newline after this line, you have removed 3 lines
thanks :)

Comment (sk9331657) [reply app/views/comments/_form.html.erb]: @avsingh999  Now is it ok??

**Comment by sk9331657** [comment]
@sagarpreet-chadha  Please review it.

**Comment by sk9331657** [comment]
@sagarpreet-chadha  @SidharthBansal  @jywarren  Please review and merge this.

**Comment by sk9331657** [comment]
@jywarren I have tested  this locally and since in case mutiple click happens , comment section will through just failure because input given by user will be empty.

**Comment by sk9331657** [comment]
@jywarren  @SidharthBansal  This PR can be merged.

**Comment by jywarren** [comment]
THANK YOU! Very exciting - looking forward to working with you!

**Comment by sk9331657** [comment]
> THANK YOU! Very exciting - looking forward to working with you!

THANK YOU! @jywarren  @SidharthBansal .

**Pull request merged**